projects/demo/src/modules/directives/copy-processor/examples/1/index.ts

```diff
@@ -0,0 +1,38 @@
+import {Component, HostListener, Inject} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiStringHandler} from '@taiga-ui/cdk';
+import {
+    NumberFormatSettings,
+    TUI_NUMBER_FORMAT,
+    TuiNotificationsService,
+} from '@taiga-ui/core';
+
+@Component({
+    selector: 'tui-copy-processor-example-1',
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    changeDetection,
+    encapsulation,
+})
+export class TuiCopyProcessorExample1 {
+    value = 12345.67;
+
+    constructor(
+        @Inject(TUI_NUMBER_FORMAT) private readonly format: NumberFormatSettings,
+        @Inject(TuiNotificationsService)
+        private readonly notificationsService: TuiNotificationsService,
+    ) {}
+
+    @HostListener('copy', ['$event'])
+    onCopy(event: ClipboardEvent) {
+        this.notificationsService
+            .show(event.clipboardData?.getData('text/plain') ?? '')
+            .subscribe();
+    }
+
+    readonly processor: TuiStringHandler<string> = text =>
```

Comment:
Doesn't lint automatically fix order on commit? @splincode

Reply:
ps: discussed it in a slack chat